modules/dhcp_common/server.rb

```diff
@@ -19,8 +19,9 @@ class Server
     include Proxy::Log
     include Proxy::Validations
 
-    def initialize(name)
-      @name    = name
+    def initialize(name, managed_subnets)
```

Comment:
This might break the API for plugins who inherit from this class, as they'll need to change their constructors. Perhaps avoid refactoring this now, or set a default value for `managed_subnets` to `Proxy::DhcpPlugin.settings.subnets`?